CRM/Contribute/Form/ContributionBase.php

```diff
@@ -1549,9 +1549,15 @@ protected function assignTotalAmounts(): void {
    */
   public function getCurrency(): string {
     $currency = $this->getContributionValue('currency');
```

Comment:
Ah - actually this is the problem - it should be` getContributionPageValue()` - not `getContributionValue()` - it's set at the page level isn't it?

We don't need the next 2 lines if we fix that

Reply:
Yes, that makes sense, thanks @eileenmcnaughton - I've updated the PR. I wondered if this was called from any other 'contribution' forms that aren't contribution pages, but I didn't notice any?